README.md

```diff
@@ -38,6 +38,11 @@ media_storage_providers:
     # Default is STANDARD.
     #storage_class: "STANDARD_IA"
 
+    # Prefix for all media in bucket, can't be changed once media has been uploaded
+    # Useful if sharing the bucket between Synapses
+    # Blank if not provided
+    #prefix: "prefix/to/files/in/bucket"
```

Comment:
Does whether a trailing slash is included here matter?

Reply:
In this case the files in the bucket would be named `prefix/to/files/in/bucketlocal_content/AB/CD/EFGH...`. A trailing slash isn't essential in anyway but visually you probably want it (or some other allowed delimiter character)